pkg/spec/spec.go

```diff
@@ -396,9 +396,9 @@ func (config *CreateConfig) createConfigToOCISpec(runtime *libpod.Runtime) (*spe
 	}
 
 	// BIND MOUNTS
-	if err := config.GetVolumesFrom(runtime); err != nil {
-		return nil, nil, errors.Wrap(err, "error getting volume mounts from --volumes-from flag")
-	}
+	// if err := config.GetVolumesFrom(runtime); err != nil {
```

Comment:
I really dislike commented out code. Since it seems to never get removed.

Reply:
Nuked it in later versions, was there as a reference to where I removed things